packages/webpack-config/src/webpack/base/module.js

```diff
@@ -34,5 +34,6 @@ export const defaultConfigModule = merge(
   useHtmlCompiler ? compileHTML : {},
   loadFonts,
   loadHandlebars,
-  copyStaticFiles ? copyStatic : {}
+  copyStaticFiles ? copyStatic : {},
```

Comment:
ich versteh jetzt nicht ganz warum es static & resources gibt. Sind das so unterschiedliche Sachen ?

Reply:
static wird direkt in target kopiert, resources in target/resources